core/src/main/java/com/venia/core/models/commerce/services/cacheinvalidation/CustomInvalidateType.java

```diff
@@ -17,20 +17,25 @@
 import com.adobe.cq.commerce.core.cacheinvalidation.spi.CacheInvalidationStrategy;
 import org.osgi.service.component.annotations.Component;
 
+import java.util.Collections;
+import java.util.List;
+
 /**
  * Strategy implementation for clearing the graphql cache based on custom attribute.
  */
 @Component(
         service = CacheInvalidationStrategy.class)
-public class CustomAttributeForClearCacheForGraphql implements CacheInvalidationStrategy {
+public class CustomInvalidateType implements CacheInvalidationStrategy {
```

Comment:
I would suggest: `CustomInvalidation`. Please provide example request Json in the Javadoc how to call this invalidation.

Reply:
I think for this I suggested CustomInvalidation.